sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3582,6 +3582,163 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
     checkAnswer(sql("SELECT 0 FROM ( SELECT * FROM B JOIN C USING (id)) " +
       "JOIN ( SELECT * FROM B JOIN C USING (id)) USING (id)"), Row(0))
   }
+
+  test("test casts pushdown on orc/parquet for integral types") {
+    def checkPushedFilters(
+      format: String,
```

Comment:
indentation? Since we have an [official code style guide](https://spark.apache.org/contributing.html), it would save your time if you read.
```
For Scala code, Apache Spark follows the official Scala style guide and Databricks Scala guide. The latter is preferred.
```
- https://github.com/databricks/scala-style-guide

Reply:
Will do. I may need to tweak my IDE so don't have to adjust by hand in future. BTW I really hope I can rely on `dev/scalafmt` for this but when I run it it generates lots of irrelevant changes. :(